crates/bevy_ecs/Cargo.toml

```diff
@@ -88,8 +88,9 @@ critical-section = [
 portable-atomic = [
   "dep:portable-atomic",
   "dep:portable-atomic-util",
-  "bevy_tasks/portable-atomic",
+  "bevy_tasks?/portable-atomic",
   "concurrent-queue/portable-atomic",
+  "spin/portable_atomic",
```

Comment:
The changes to `bevy_ecs/Cargo.toml` _could_ be split into their own PR, but for now I'm just including them here for brevity. I'm happy to split it out if anyone has any concerns.